spyglass/cli.py

```diff
@@ -38,11 +38,13 @@ def main(args=None):
         parsed_args.lensposition,
         parse_autofocus_speed(parsed_args.autofocusspeed))
 
+    print('Available controls:\n'+str(picam2.camera_controls))
```

Comment:
Would it make sense to at an option to list all available controls and then exit? I don't think we need to list them each time spyglass starts

Reply:
ofc we should add a cli argument. This was only a quick dirty way on a WIP branch to quickly see all of the controls.